pkg/controllers/servicescale/scaler.go

```diff
@@ -265,6 +265,8 @@ func (s *SimpleScale) scrape() error {
 		metricURL := fmt.Sprintf("http://%s:4191/metrics", pod.Status.PodIP)
 		responseTotalMatchCriteria := fmt.Sprintf("response_total{authority=\"%s-%s.%s.svc.cluster.local", s.app, s.version, s.namespace)
 		requestTotalMatchCriteria := fmt.Sprintf("request_total{authority=\"%s-%s.%s.svc.cluster.local", s.app, s.version, s.namespace)
+		inbound := "direction=\"inbound\""
+		outbound := "direction=\"outbound\""
```

Comment:
Nit, but this feels weird. What if you make inbound a `bool` param on `calculateActiveRequests` and if true use the inbound string. Or you could use const.